libpod/container.go

```diff
@@ -554,10 +555,22 @@ func (c *Container) Init() (err error) {
 		}
 	}
 
+	runDirResolv := filepath.Join(c.state.RunDir, "resolv.conf")
+
+	// Copy /etc/resolv.conf to the container's rundir
+	if err := c.copyResolvToRunDir(); err != nil{
+		return errors.Wrapf(err, "unable to copy /etc/resolv.conf to ", runDirResolv)
+	}
+	// Relabel runDirResolv for the container
+	label.Relabel(runDirResolv, c.config.MountLabel, true)
```

Comment:
I feel like this should be in copyResolvToRunDir() - setting the appropriate permissions feels like part of copying

Reply:
good suggestion, done